app/src/js/config/config.js

```diff
@@ -9,6 +9,7 @@ const config = {
     }
   },
   apiRoot: process.env.APIROOT || 'https://example.com',
+  awsRegion: process.env.AWS_REGION || 'us-west-2',
```

Comment:
ummm... I don't think we can assume this is the default to use. Does it matter what is here?

Reply:
Why can't we assume that as a default?  A user would just override with whatever they have set.  And I'm pretty sure most daacs are being launched into us-west-2